sql/rpl_rli_pdb.cc

```diff
@@ -1435,18 +1435,14 @@ ulong circular_buffer_queue::en_queue(void *item)
   return ret;
 }
 
-void* circular_buffer_queue::head_queue()
+uchar* circular_buffer_queue::head_queue()
```

Comment:
The change is correct but it's a cleanup instead of a part of a fix - I see that the result value is cast anyway? Or is it required? If it's a cleanup only, please remove, it adds to our maintenance diff